cmd/tx.go

```diff
@@ -898,7 +898,18 @@ $ %s tx raw send ibc-0 ibc-1 100000stake cosmos1skjwj5whet0lpe65qaq4rpq03hjxlwd9
 
 			// Query all channels for the configured connection on the src chain
 			srcChannelID := args[4]
-			channels, err := src.ChainProvider.QueryConnectionChannels(cmd.Context(), srch, path.Src.ConnectionID)
+			srcChainID := args[0]
+
+			var pathConnectionID string
+			if srcChainID == path.Src.ChainID {
+				pathConnectionID = path.Src.ConnectionID
+			} else if srcChainID == path.Dst.ChainID {
+				pathConnectionID = path.Dst.ConnectionID
+			} else {
+				return fmt.Errorf("no path configured using chain-id: %s", src.Chainid)
```

Comment:
Same thing here change `src.Chainid` to  `src.ChainID()`

Reply:
you are right, I agree with you.
But it doesn't work as intended.
I'm looking for the cause now. I registered as an issue below.